### ISSUE TITLE
Messaging: Deprecate runEveryX apps

### ISSUE THREAD

**Issue by ivanagas** [PostHog/meta#202]
# Messaging

### Context

[Pipeline is deprecating apps](https://posthog.slack.com/archives/C0374DA782U/p1706793144391539?thread_ts=1706639141.395339&cid=C0374DA782U) that are basically running cron jobs for people, aka runEveryX apps. These include:

- [Github star sync](https://github.com/PostHog/github-star-sync-plugin)
- [Github release tracker](https://github.com/PostHog/github-release-tracking-plugin)
- [Shopify Plugin](https://github.com/PostHog/posthog-shopify-sync-plugin)
- [Orbit Love sync](https://github.com/PostHog/posthog-orbit-love-plugin)
- [GitLab Release tracker](https://github.com/PostHog/gitlab-release-tracking-plugin)
- [Ingestion alert plugin](https://github.com/PostHog/ingestion-alert-plugin)
- [AWS Kinesis](https://github.com/PostHog/posthog-kinesis-plugin)
- [PagerDuty plugin](https://github.com/PostHog/posthog-pagerduty-plugin)

We documented the alternative to doing this (running your own, using the warehouse) here: https://posthog.com/docs/cdp/common-questions#how-do-i-capture-data-from-another-application

We need to tell the people using these to migrate.

### Goal

People stop using the apps and we can deprecate without them getting mad.

### Format

Email to customers.

### Audience / Targeting

These organizations in [NA](http://metabase-prod-us/question/367-apps-full-overview-methods-tasks-jobs) and [EU](http://metabase-eu/question/120-apps-full-overview-methods-tasks-jobs)

### Resources, timeline, responsibilities

Single message

- [ ] Write message - @ivanagas and @tiina303 
- [ ] Send message - @joethreepwood 

### Previews

We are deprecating the functionality of PostHog's data pipelines to capture events from other services. This includes the following apps:

- Github star sync
- Github release tracker
- Shopify Plugin
- Orbit Love sync
- GitLab Release tracker
- Ingestion alert plugin
- AWS Kinesis
- PagerDuty plugin

You currently have at least one of these apps running, and once we deprecate it, it will not work. To avoid unexpected changes, there are two options for maintaining this functionality:

1. **Set up a cron of your own.** You can use a tool like [val.town](https://val.town/) to easily set up scheduled tasks that capture data from other tools and send them to PostHog. Our tutorials on [capturing new RSS items](https://posthog.com/tutorials/rss-item-capture), [events from Calendly webhooks](https://posthog.com/tutorials/calendly-webhooks), and [GitHub stars](https://posthog.com/tutorials/github-star-tracker) are examples of this.

2. **Use our data warehouse.** PostHog's data warehouse enables you to import and use data from platforms like Stripe, Hubspot, and Postgres natively in PostHog. See our [docs](https://posthog.com/docs/data-warehouse) for more information.

Event capture from the API or SDKs is unchanged. You can find [more details in our docs](https://posthog.com/docs/cdp/common-questions#how-do-i-capture-data-from-another-application).

**Comment by tiina303** [comment]
Given that we're linking to data warehouse it might be better to send out this message once they have released publicly (which I'm hoping will be soon)?

**Comment by joethreepwood** [comment]
> These organizations in [NA](http://metabase-prod-us/question/367-apps-full-overview-methods-tasks-jobs) and [EU](http://metabase-eu/question/120-apps-full-overview-methods-tasks-jobs)

These links aren't working, and I don't think I have Metabase access. 

Can someone either point me to a cohort for users in these orgs, or else give me an export of Org_IDs that I can use?

**Comment by joethreepwood** [comment]
<img width="739" alt="Screenshot 2024-05-01 at 12 06 04" src="https://github.com/PostHog/meta/assets/84011561/12ae924d-26f4-47d7-8b70-e69b5f04d48e">

Some copy changes for tightness and clarity. 

We need to set a deadline for this. Ideally, it'd be at least two weeks?

**Comment by ivanagas** [comment]
For "set up a cron of your own," we can adjust slightly to give more context:

> **Set up a cron of your own.** These apps were functionally a cron job PostHog was running for you. You set one up yourself to schedule data capture tasks using a tool like AWS Lambda, Cloudflare Workers, or Val.town. Our tutorials...

Two weeks after email seems good with me.

@tiina303 can you help with cohorts or org_id list?

**Comment by tiina303** [comment]
Data in https://posthog.slack.com/archives/C0374DA782U/p1714611383781519?thread_ts=1706639141.395339&cid=C0374DA782U to avoid sharing details in a public issue

**Comment by joethreepwood** [comment]
<img width="1267" alt="Screenshot 2024-05-02 at 09 53 37" src="https://github.com/PostHog/meta/assets/84011561/46d6a23c-0986-4b69-b63a-c4cfabc2caa9">

Final copy.

I've set 20th May as a date, meaning I'll send this email on Monday morning. If we send earlier then the deadline overlaps with the offsite when we may not be able to best support users, and it's generally bad to send urgent comms on a Friday anyway.

**Comment by simfish85** [comment]
I think we need to do something specific here for Shopify as a lot of people will be using that particular one for revenue tracking within PostHog.  The generic guidance isn't enough here so can we get a step by step tutorial for that one in particular?  Also maybe target the messaging for them.

**Comment by charlescook-ph** [comment]
How many of the people affected are CS-managed customers? There could be an opportunity for a separate message where we can try to drive a bit of data warehouse adoption (don't let this be a blocker though, but worth taking a quick look?)

**Comment by simfish85** [comment]
Good point - those metabase queries are really hard to parse to answer that question.  Can we get a list of all orgs using each of the affected plugins?  Then I'll go through and check for CS managed.

**Comment by joethreepwood** [comment]
Org_IDs here: https://docs.google.com/spreadsheets/d/183lL5BoGGwEcHnb6P4LPhtLufSqbBIUfUkdUDsjfS6w/edit?usp=sharing

I'm setting up cohorts now.

**Comment by joethreepwood** [comment]
There's a total of 9 US teams using the Shopify plugin, all in the US. 

I don't think it's worth setting up any specific content for a group this small -- it'd be better to handle them personally if they're CS priorities.

**Comment by joethreepwood** [comment]
I've got the segment set up and scheduled, as per below. I don't think we need to target this much more, but there's time to disagree as I don't have a strong feeling on it. 

Note that we did have 22 errors in the segment creation -- 22 people who were attached to the org, but whom weren't already in Customer.io. This is probably a quirk of the long-standing Customer.io integration issues. I've manually created and added these users to be sure they get the emails. 

All data available here: 
https://docs.google.com/spreadsheets/d/183lL5BoGGwEcHnb6P4LPhtLufSqbBIUfUkdUDsjfS6w/edit?usp=sharing

<img width="594" alt="Screenshot 2024-05-02 at 10 58 12" src="https://github.com/PostHog/meta/assets/84011561/c5bdd275-8c5f-4042-aab5-1308263e573c">

**Comment by simfish85** [comment]
> There's a total of 9 US teams using the Shopify plugin, all in the US.

Fair enough - thought it would be way more widely used than that

Just had a look at the orgs, nothing in there that needs special CS attention

**Comment by joethreepwood** [comment]
> Fair enough - thought it would be way more widely used than that

It's because these aren't set to global, I think -- so they haven't been available for new users to try for a fair ol' while.

**Comment by tiina303** [comment]
Looked at the customer priorities for other apps 
There are 2 changes we should do (to the text & who gets the emails), that I called out in [slack](https://posthog.slack.com/archives/C0374DA782U/p1714611383781519?thread_ts=1706639141.395339&cid=C0374DA782U) already: 
1. exclude AWS Kinesis and PagerDuty - we'll white glove these
2. Include Zendesk (missing from the plugins list in the email)
See https://docs.google.com/spreadsheets/d/15GORbTZ8fX6UVM12s3qM9gsbo5AXaEdVo8X_FCxwAEc/edit#gid=978899497
comments for more details

**Comment by joethreepwood** [comment]
Unfortunately, due to me being off ill on Friday and the Monday bank holiday, this email has already been sent as above. Tiina's feedback came in at a good time and would normally have been picked up, but I was laid low on Friday with a migraine and didn't get the feedback until today -- very sorry about that!

For 1) I don't think there's much else we should do here. The relevant team can reach out and white-glove this as planned -- at worst, this is a value add for users. 

For 2) I'll prepare a second, dedicated email which I'll share below shortly focused on just the impacted users.

**Comment by joethreepwood** [comment]
<img width="776" alt="Screenshot 2024-05-07 at 11 50 45" src="https://github.com/PostHog/meta/assets/84011561/5ec47f64-192a-41a3-801b-bf30a4745ec9">

@tiina303 After a bit of research, this would seem a viable recommendation for the one affected customer -- and also means that we wouldn't have to handle support as the ownership exists with Zendesk and AWS. If you agree, I can schedule this today.

**Comment by joethreepwood** [comment]
Confirmed via Slack, and sent the above with a CTA that directs to Pipeline support.
